workday/13.0/modules/ROOT/pages/workday-connector-studio.adoc

```diff
@@ -112,9 +112,9 @@ Enter the following information on the *General* tab of the *Global Element Prop
 [%header,cols="30s,70a"]
 |===
 |Field |Description
-|Connection |Select `Basic authentication api v34`.
+|Connection |Select `Basic authentication`.
 |Password |Enter the password with which to log in.
-|Host Name |Enter the host name of one of Workday’s Cloud Servers. Default value: `impl-cc.workday.com`.
+|Host Name |Enter the host name of one of Workday’s Cloud Servers.
```

Comment:
```suggestion
|Host Name |Enter the host name of a Workday cloud server.
```
I performed an internet search for "workday cloud server" and could not find it anywhere init. capped (although Workday Cloud Platform (WCP) is).